ui/src/app/navigation/commoncontrols.component.ts

```diff
@@ -14,10 +19,26 @@ export class CommoncontrolsComponent extends AbstractSearchComponent implements
   searchIcon = SvgHelper.path(SvgIcon.SEARCH)
   dismissIcon = SvgHelper.path(SvgIcon.DISMISS)
 
-  constructor(protected searchService: SearchService, protected route: ActivatedRoute, protected authService: AuthService) {
+  selectedFilters: Set<PublishStatus> = new Set([PublishStatus.Draft, PublishStatus.Published])
```

Comment:
@JohnKallies  I put the logic in common controls because the button lives inside the template with the same name. We could put the logic in a parent component or the abstract search component that it extends from?